rancid-tomatillos/src/App.js

```diff
@@ -1,26 +1,32 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, {Component} from "react";
+import MainView from "./components/MainView";
+import CardsContainer from "./components/CardsContainer";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import "./App.css";
+
+class App extends Component {
+    constructor() {
+        super();
+        this.state = {
+            movies: null
+        };
+    }
+    componentDidMount() {
```

Comment:
This was a fun win, figuring out how to fetch movies from the API and have `App`  pass down as a prop to children components!